lib/experimental/Information/Headers/BaseHeader/index.tsx

```diff
@@ -110,6 +130,10 @@ export function BaseHeader({
     }
   }, [measureSize.height, descriptionSize.height])
 
+  const primaryActionDropdown = primaryAction as
```

Comment:
do not cast things, that's a code smell. Prefer to create a new type and make the shift to the new one explicit